packages/web-app-files/src/components/SideBar/Shares/Collaborators/InviteCollaborator/InviteCollaboratorForm.vue

```diff
@@ -98,11 +98,20 @@ export default {
     RecipientContainer,
     ExpirationDatepicker
   },
+  props: {
+    savingButtonTitle: {
+      type: String,
+      required: false,
+      default: 'Share'
```

Comment:
Needs to be a function and wrapped with gettext, i.e. `default: () => this.$gettext("Share")`.

Reply:
Its already translated via markup, using it like this throws errors at unittests

Reply:
what do you mean by `translated via markup`? I see no static string extraction in the template.

Reply:
Sorry, meant in the template. One is using v-translate and the other $gettext.

Reply:
`v-text="$gettext(saveButtonLabel)"`

Reply:
well, string extraction for transifex doesn't interpret code, it only reads static content. i.e. `$gettext(someVariable)` is not usable for string extraction. It only works if you have the content in some other place already where it's wrapped with `$gettext` (or uses on of the other easygettext translation variations).